### DISCUSSION TITLE
Is the current browse icon correct?

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#19884] [Q&A] [answered]
I presume it corresponds rather with "download" than "browse". Should it be changed to smth with magnifier? 

![image](https://user-images.githubusercontent.com/86934170/185787611-1dbacd1a-cf02-4cc0-a2d6-aba740f9b961.png)

**Comment by peppy** [comment, marked as answer]
All the icons need to be redone. Using font awesome everywhere is shoddy.
